packages/@aws-cdk/aws-kinesisfirehose-alpha/lib/source.ts

```diff
@@ -0,0 +1,79 @@
+import { Construct } from 'constructs';
+import { CfnDeliveryStream } from 'aws-cdk-lib/aws-kinesisfirehose';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import * as kinesis from 'aws-cdk-lib/aws-kinesis';
+
+/**
+ * A Kinesis Data Firehose delivery stream source configuration.
+ *
+ * @internal
+ */
+export interface SourceConfig {
```

Comment:
Is it necessary to export the interface if we're only going to use it internally? At a glance, it doesn't seem like it's being used outside of this file.

Reply:
Good catch, originally I was using it as a return in the delivery-stream but removed that since. This probably does not need to be exported anymore. Will make the change.